sdk/java-sdk-spring/src/it/java/com/example/wiring/pubsub/DockerIntegrationTest.java

```diff
@@ -49,25 +49,15 @@ public abstract class DockerIntegrationTest {
   protected WebClient webClient;
   protected Duration timeout = Duration.of(5, SECONDS);
 
-  private DockerComposeUtils dockerComposeUtils = new DockerComposeUtils("docker-compose-integration.yml");
 
   private KalixSpringApplication kalixSpringApplication;
 
   public DockerIntegrationTest(ApplicationContext applicationContext) {
     Map<String, Object> confMap = new HashMap<>();
-    confMap.put("kalix.user-function-port", dockerComposeUtils.userFunctionPort());
     // don't kill the test JVM when terminating the KalixRunner
     confMap.put("kalix.system.akka.coordinated-shutdown.exit-jvm", "off");
-    // dev-mode should be false when running integration tests
-    confMap.put("kalix.dev-mode.enabled", false);
```

Comment:
we have been copying this over and over, but there is no such a thing as `dev-mode.enabled`. We probably had it in the past, but it's gone.